docs/utilities/batch.md

```diff
@@ -141,12 +141,23 @@ Processing batches from SQS works in three stages:
 
 #### FIFO queues
 
-When using [SQS FIFO queues](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/FIFO-queues.html){target="_blank"}, we will stop processing messages after the first failure, and return all failed and unprocessed messages in `batchItemFailures`.
-This helps preserve the ordering of messages in your queue.
+When using [SQS FIFO queues](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-fifo-queues.html){target="_blank"}, a batch may include messages from different group IDs.
 
-```typescript hl_lines="1-4 8 20-22"
---8<-- "examples/snippets/batch/gettingStartedSQSFifo.ts"
-```
+By default, we will stop processing at the first failure and mark unprocessed messages as failed to preserve ordering. However, this behavior may not be optimal for customers who wish to proceed with processing messages from a different group ID.
+
+Enable the `skipGroupOnError` option for seamless processing of messages from various group IDs. This setup ensures that messages from a failed group ID are sent back to SQS, enabling uninterrupted processing of messages from the subsequent group ID.
+
+=== "Recommended"
+
+    ```typescript hl_lines="1-4 13 28-30"
```

Comment:
Here we should change the hl_lines to 1-4 and 8, otherwise it'll look like this

<img width="856" alt="Screenshot 2024-06-03 at 16 56 10" src="https://github.com/aws-powertools/powertools-lambda-typescript/assets/7353869/42892cf2-351c-4064-9407-569a04ac3ac5">